dev/Repeater/ViewManager.cpp

```diff
@@ -298,6 +298,8 @@ void ViewManager::UpdatePin(const winrt::UIElement& element, bool addPin)
                     repeater.InvalidateMeasure();
                 }
             }
+
+            break;
```

Comment:
This is fine, but might be clearer as a two step process. One quick loop to find the first parent who is an ItemsRepeater. And then a second step (outside the loop) that if one was found performs the remaining pin logic.